hosted-tenant-base/pom.xml

```diff
@@ -235,12 +235,11 @@
                             <!-- note: ordering affects how overrides are evaluated! -->
                             <testBundleScopeOverrides>
                                 ${extraTestBundleScopeOverrides},
-                                com.yahoo.vespa:application:test,
-                                com.yahoo.vespa:container-test:runtime,
                                 org.junit.jupiter:junit-jupiter-api:provided,
-                                org.opentest4j:opentest4j:test,
                                 org.junit.jupiter:junit-jupiter-engine:test,
                                 org.junit.vintage:junit-vintage-engine:test,
+                                com.yahoo.vespa:application:test,
+                                com.yahoo.vespa:container-test:runtime,
```

Comment:
Moved these down, in case they ever end up exposing the above dependencies, as that would shadow those (silently).